### PR TITLE
Patch 6

### PR DESCRIPTION
**Pull request by Ciao287**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/ciaobot.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   },
   "record": {
     "A": ["144.91.115.195"],
-    "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
-    "TXT": ["v=spf1 include:spf.improvmx.com ~all", "google-site-verification=ESHkpD4wk4-a2cqONn73cRaELmis2IzdBRB3Fw-t1Dw"]
+    "MX": "mail.ciaohost.tech",
```
Comment (Stef-00012) [inline domains/ciaobot.json:8]: MX deve essere un array

File: domains/_autodiscover._tcp.ciaobot.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Ciao287",
+    "discord": "687333016921440317"
+  },
+  "record": {
+    "SRV": "mail.ciaohost.tech 443"
```
Comment (Stef-00012) [inline domains/_autodiscover._tcp.ciaobot.json:7]: i record SRV non so formattati così, vedi https://is-a.dev/docs/domain-structure/

**Review by Stef-DP — CHANGES_REQUESTED** [review]
^

File: autodiscover.ciaobot.json
Comment (Stef-00012) [inline autodiscover.ciaobot.json:1]: deve essere nella cartella domains

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by Stef-DP — APPROVED** [review]

**Pull request merged**